metabase/metabase
# Remove transition between modals in events

Epic: https://github.com/metabase/metabase/issues/20289

Changes:
- Adds `enableTransition` to `Modal` component to prevent modal transitions for event modals
- Adds custom `LoadingAndErrorWrapper` to set `min-height` for event models when data is loading

How to test:
- Go to `/collection/root`
- Click on the calendar icon in the collection header
- Create several events
- Edit an event
- Make sure there was no sliding animation when navigating between modals
